config.go

```diff
@@ -102,6 +102,8 @@ type config struct {
 	DBFileName         string `long:"dbfile" description:"SQLite DB file name (default is dcrdata.sqlt.db)." env:"DCRDATA_SQLITE_DB_FILE_NAME"`
 	AgendaDBFileName   string `long:"agendadbfile" description:"Agenda DB file name (default is agendas.db)." env:"DCRDATA_AGENDA_DB_FILE_NAME"`
 
+	PurgeNBestBlocks int `long:"purge-n-blocks" description:"Purge all data for the N best blocks, using the best block across all DBs if they are out of sync."`
```

Comment:
To be clear, this means that if PG is at 1000 and SQLite is at 950, and `purge-n-blocks` is 50, then blocks 951 to 1000 in PG will be purged and SQLite will be untouched.

Reply:
But this PR only implements rewind of PG.